server/Cargo.toml

```diff
@@ -22,6 +22,7 @@ dotenv = { version = "0.15.0", features = ["clap"] }
 opentelemetry = { version = "0.18.0", features = ["trace", "rt-tokio", "metrics"] }
 opentelemetry-prometheus = "0.11.0"
 prometheus = { version = "0.13.3", features = ["process"] }
+redis = "0.22.3"
```

Comment:
We may want to lock this behind a compiler flag in future so we can produce smaller binaries. Small thing